docs/api/workflows/steps/graph.md

```diff
@@ -0,0 +1,4 @@
+::: cognite.neat.workflows.steps.lib.graph_store
+::: cognite.neat.workflows.steps.lib.graph_extractor
+::: cognite.neat.workflows.steps.lib.graph_loader
+::: cognite.neat.workflows.steps.lib.graph_transformer
```

Comment:
Is it worth having some extra headers inbetween the modules to make navigation easier.